src/unix/proctitle.c

```diff
@@ -98,7 +98,10 @@ int uv_get_process_title(char* buffer, size_t size) {
   else if (size <= process_title.len)
     return -ENOBUFS;
 
-  memcpy(buffer, process_title.str, process_title.len + 1);
+  if (process_title.len != 0) {
+    memcpy(buffer, process_title.str, process_title.len + 1);
+  }
```

Comment:
Tiniest of nits but the libuv house style is to omit braces for single if statements.